core/includes/bootstrap.inc

```diff
@@ -1463,7 +1463,7 @@ function state_del($name) {
  * @since 1.30.0 Function added.
  */
 function backup_get_backup_directory() {
-  $backup_directory = settings_get('backup_directory');
+  $backup_directory = settings_get('backup_directory', FALSE);
```

Comment:
`FALSE` is intended to have special treatment. An empty string or NULL is the equivalent of "not configured" while `FALSE` means "intentionally disabled".

```suggestion
  $backup_directory = settings_get('backup_directory', '');
```

Reply:
I've merged the suggestion